Sources/WWDCNotes/WWDCNotes.docc/WWDC24/WWDC24-10166-Build-compelling-spatial-photo-and-video-experiences.md

```diff
@@ -8,11 +8,113 @@ Learn how to adopt spatial photos and videos in your apps. Explore the different
    @CallToAction(url: "https://developer.apple.com/wwdc24/10166", purpose: link, label: "Watch Video (21 min)")
 
    @Contributors {
-      @GitHubUser(<replace this with your GitHub handle>)
+      @GitHubUser(halmueller)
    }
 }
 
-😱 "No Overview Available!"
+## Chapters
+ [0:00 - Introduction](https://developer.apple.com/videos/play/wwdc2024/10166/?time=0)  
+ [1:07 - Types of stereoscopic experiences](https://developer.apple.com/videos/play/wwdc2024/10166/?time=67)  
+ [4:13 - Tour of the new APIs](https://developer.apple.com/videos/play/wwdc2024/10166/?time=253)  
+ [13:14 - Deep dive into spatial media formats](https://developer.apple.com/videos/play/wwdc2024/10166/?time=794)  
+[21:16 - Wrap-up](https://developer.apple.com/videos/play/wwdc2024/10166/?time=1276)  
+
+### Resources
+
+- AVCam
+- Converting side-by-side 3D video to multiview HEVC and spatial video
+- Creating spatial photos and videos with spatial metadata
+- Forum: Spatial Computing
+- Writing spatial photos
+- HD Video SD Video
+
+### Related Videos:
```

Comment:
Related sessions are part of the metadata and get added automatically during deployment, see:
https://wwdcnotes.github.io/WWDCNotes/documentation/wwdcnotes/contributing/#Image-Name--Format

This matches up exactly with the related sessions you provided.

<img width="805" alt="Bildschirmfoto 2024-06-11 um 12 23 52 AM" src="https://github.com/WWDCNotes/WWDCNotes/assets/6942160/acfef732-156c-487b-977d-d70240136f90">

So I think there's no need for this section. Please just remove it.

Reply:
Thanks for that! I was not looking forward to manually adding a dozen links. I think I have resolved this.